infrastructure/networking.tf

```diff
@@ -2,7 +2,7 @@
 # related to networking.
 
 provider "aws" {
-  version = "1.37.0"
+  version = "2.70.0"
```

Comment:
also just check that this is intended

Reply:
This newer version of terraform didn't support this version of the AWS provider. `terraform 0.12-checklist` recommended version 3.12.0 which is the latest version, but that didn't support such an old version of terraform. This was the latest version starting with `2.` and `terraform 0.12-checklist` no longer recommended a later provider version.